pandas/core/arrays/boolean.py

```diff
@@ -153,6 +154,18 @@ def __from_arrow__(
         else:
             return BooleanArray._concat_same_type(results)
 
+    def _get_common_dtype(self, dtypes: list[DtypeObj]) -> DtypeObj | None:
+        # Handle only boolean + np.bool_ -> boolean, since other cases like
+        # Int64 + boolean -> Int64 will be handled by the other type
+        if all(
+            isinstance(t, BooleanDtype)
```

Comment:
is_bool_dtype should work here no? 

that said i think we are using mostly this type of pattern, which i am actually not much of a fan of. so ok for this PR. but we should ideally be using the common accessors rather than writing new ones for EA types, can you open an issue about this.

Reply:
Ok, will do